src/app/wiki/wiki.component.html

```diff
@@ -0,0 +1,127 @@
+      <md-list *ngIf="searchResults && searchResults.length">
+        <wiki-list-entry
+            *ngFor="let video of searchResults"
+            (update)="update()"
+            [video]="video"
+            [admin]="admin"
+        >
+        </wiki-list-entry>
+      </md-list>
+      <span *ngIf="!searchResults || !searchResults.length" class="notfound">no videos found</span>
+    </div>
+  </div>
+
+  <!-- content -->
+  <div *ngIf="!searching">
+
+    <!-- categories/topics -->
+    <div class="section">
+      <h4 (click)="showCategory = !showCategory">
+        <md-icon>{{ showCategory ? 'keyboard_arrow_down' : 'keyboard_arrow_right' }}</md-icon>
+        <span>select topic</span>
+        <span *ngIf="selectedCategory" class="selectedCategory"> ({{ selectedCategory }})</span>
+      </h4>
+      <div [hidden]="!showCategory">
+        <md-spinner *ngIf="loadingCategory" class="load-more-spinner"></md-spinner>
+        <!-- list of categories -->
+        <ul *ngIf="categories">
+          <li *ngFor="let cat of categories"
+              (click)="changeCategory(cat)"
+              [class.selected]="cat === selectedCategory">
+            {{ cat }} ({{ data[cat]['count'] }})
+          </li>
+        </ul>
+        <span *ngIf="!categories" class="notfound">no categories found</span>
+      </div>
+      <md-divider></md-divider>
+    </div>
+
+
+    <!-- tags -->
+    <div class="section">
+      <h4 (click)="showTags = !showTags">
+        <md-icon>{{ showTags ? 'keyboard_arrow_down' : 'keyboard_arrow_right' }}</md-icon>
+        <span>select tags</span>
+      </h4>
+      <div [hidden]="!showTags">
+        <div *ngIf="selectedCategory && data">
+          <md-spinner *ngIf="loadingTags" class="load-more-spinner"></md-spinner>
+          <!-- toggle all tags buttons -->
+          <div>
+            <md-icon title="disable all" (click)="toggleAllTags(false)">visibility_off</md-icon>
+            <md-icon title="enable all" (click)="toggleAllTags(true)">visibility</md-icon>
+          </div>
+          <!-- toggle single tag buttons -->
+          <div *ngIf="tags" class="tags">
+            <div *ngFor="let tag of tags"
+                 class="tag"
+                 [class.active]="selectedTags[tag]"
+                 (click)="toggleTag(tag)">
+              {{ tag }}
+            </div>
+          </div>
+        </div>
+        <span *ngIf="!selectedCategory" class="notfound">no category selected</span>
+      </div>
+      <md-divider></md-divider>
+    </div>
+
+
+    <!-- selected videos -->
+    <div class="section">
+      <h4 (click)="showVideos = !showVideos">
+        <md-icon>{{ showVideos ? 'keyboard_arrow_down' : 'keyboard_arrow_right' }}</md-icon>
+        <span>select video</span>
+      </h4>
+      <div [hidden]="!showVideos">
+        <md-spinner *ngIf="loadingVideos" class="load-more-spinner"></md-spinner>
+        <md-list *ngIf="selectedVideos">
+          <wiki-list-entry
```

Comment:
@binarious I wasn't able to remove the `padding-left` from the `md-list-item`'s. On small screens it causes most of the title to disappear.